docs/articles/workflow/creating-release-process.rst

```diff
@@ -0,0 +1,56 @@
+:orphan:
+
+Creating a release process
+==========================
+
+In order to make sure the software that one releases is of high quality,
+traceable, reproducible and so on, it is important to have a well-defined
+release process. We describe a simple process for how to create a release
+process.
+
+It is important to document the process and the various steps in it. In order to
+be able to improve the actual process, and to keep it transparent we suggest
+that the release process is kept together with the rest of the documentation for
+the project.
+
+Create a list of artifacts
+--------------------------
+To be able to maintain a release, a list of artifacts is required to have. The
```

Comment:
I don't understand this first sentence. What do you wish to communicate?

Reply:
That it is important to know what artifacts are in a release. But I agree that it can be written in a nicer way.